src/Framework/FieldsAPI/Amount.php

```diff
@@ -18,6 +18,15 @@ class Amount extends Field
      */
     protected $allowCustomAmount = false;
 
+    public function __construct($name)
+    {
+        parent::__construct($name);
+
+        $this->validationRules->rule('numeric', true);
+        $this->validationRules->rule('integer', true);
+        $this->required();
```

Comment:
I was really on the fence about this, @jonwaldstein, and would like your thoughts. The alternative is to apply all validation rules during the conversion process.

Reply:
Ah yeah, I was wondering where the validation rules for the amount field went.  I think for consistency, I would prefer these to be added during the conversion process - especially required.  If we start using this field in other places and it's always going to be the same validation rules then maybe i can see this approach valuable as an abstraction - but so far other fields don't have them baked in.

Reply:
I can't move them all over to the conversion, yet, as the validationRules property is protected in core. I want to think through that a bit more, too. So I've at least put required back in f45d2f7.